ocfweb/lab_reservations/templates/lab_reservations/index.html

```diff
@@ -0,0 +1,92 @@
+{% extends "base.html" %}
+{% load bootstrap %}
+{% load staticfiles %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-md-8 ocf-content-block">
+
+            <form method="post" action="{% url 'request_reservation' %}">
+                {% csrf_token %}
+                <p>
+                    This form is <strong>only</strong> for student groups desiring to
+                    use the OCF's lab space. If you want to create an individual account
+                    please visit <a href="{% url 'register' %}">this link</a>. If you
+                    want to request an account for a student group, please visit
+                    a staff member in the OCF lab during
+                    <a href="{% url 'staff-hours' %}">staff hours</a>.
+                </p>
+
+                <hr />
+
+                <p>{{form.real_name|bootstrap}}</p>
+
+                <hr />
+                Please provide a UC Berkeley e-mail address. We will use this
+                to send you a confirmation email in addition to any necessary
+                follow-ups (including an approval email).
+                <p>{{form.contact_email|bootstrap}}</p>
+                <p>{{form.verify_contact_email|bootstrap}}</p>
+
+                <hr />
+
+                <p>{{form.student_group|bootstrap}}</p>
+                <p>{{form.reason|bootstrap}}</p>
+
+                <hr />
+
+                <p>{{form.date|bootstrap}}</p>
+                <p>{{form.starttime|bootstrap}}</p>
+                <p>{{form.endtime|bootstrap}}</p>
+
+
+                <div class="well">
+                    <p><strong>
+                        Please read our lab reservation policy
+                        <a href=https://www.ocf.berkeley.edu/docs/services/lab/lab-reservation-policy/>
```

Comment:
missing quotes on the href attribute

also, please don't do absolute links like this, use the `{% url %}` template helpers -- this helps ensure all links are up-to-date and that we don't have any broken links (they would fail at test time and never make it into production)